### PR TITLE
Introducing ThingSpeak webhook template

### PR DESCRIPTION
**Pull request by nejraselimovic**
#### Summary
Adding ThingSpeak webhook template.

#### Checklist
- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible, they don't break existing deployments.
- [x] Testing: The changes are tested.
- [ ] Documentation: Relevant documentation is added or updated.

### CODE REVIEW

**Comment by nejraselimovic** [comment]
@johanstokking adding you as a reviewer since Adrian is off

File: thingspeak.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+fields:
+  - id: channelid
+    name: Channel ID
+    description: ThingSpeak Channel ID
+    secret: false
+    default-value: 
+  - id: apikey
+    name: API Key
+    description: ThingSpeak Write API Key
+    secret: false
```
Comment (johanstokking) [inline thingspeak.yml:16]: Isn't this a secret?

File: thingspeak.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    default-value: 
+format: json
+headers:
+  Content-Type: application/json
+  Authorization: "{apikey}"
+  ChannelId: "{channelid}"
```
Comment (johanstokking) [inline thingspeak.yml:22]: Idiomatic way for formatting HTTP header keys would be `Channel-Id`. Sure it's `ChannelId`?

File: thingspeak.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+create-downlink-api-key: false
+base-url: http://api.thingspeak.com/things_network/v3/update
```
Comment (johanstokking) [inline thingspeak.yml:24]: I really hope this is `https` enabled?

File: thingspeak.yml (hunk 2 above)
Comment (nejraselimovic) [reply thingspeak.yml:22]: Yes, I tested this now once again, and it doesn't work with `Channel-Id`.

**Review by johanstokking — APPROVED** [review]

**Pull request merged**